github-actions/scripts/review_release/main.py

```diff
@@ -0,0 +1,84 @@
+"""
+Determines whether a given app release requires manual review or can be automatically released.
+"""
+import logging
+import os
+import re
+import sys
+
+from github import Github
+
+from ..common import get_app_json
+
+COMMIT_STATUS_PATTERN = re.compile(r'AWS CodeBuild (BuildBatch )?us-west-[1-2] \((?P<context>.+)\)')
+REQUIRED_COMMIT_STATUSES = [
+    'prodsec-scans',
+    'build-app',
+    'static-tests',
+    'integration-tests',
+    'sanity-tests'
+]
+
+
+def review_release(app_repo, app_dir_path, release_git_ref):
+    # Require manual approval for Splunk-supported apps
+    manual_approval = False
+    app_json = get_app_json(app_dir_path)
+    if app_json['publisher'] == 'Splunk':
```

Comment:
kind of an edge case but if someone changes the publisher would it allow them to bypass this when publishing? or would something else block that? I'm thinking we would want a manual approval in general if the publisher ever changes to/from Splunk (certification/decertification) aside from the certified case

Reply:
good idea, I added a check to require manual approval if we're switching from `non-splunk supported --> splunk-supported` or `splunk-supported --> non-splunk-supported`